### DISCUSSION TITLE
about packages

### DISCUSSION THREAD

**Discussion by MatinJiridity** [semaphore-protocol/semaphore#143] [General]
I wish you define every package, special circuits 
I will be happy if you explain more details about circuits how it works and where we need it...

**Comment by cedoor** [comment]
> I wish you define every package, special circuits

Hey, we'll create a README file for our circuits very soon. Do you think we should add more information to the other packages' README files?

> I will be happy if you explain more details about circuits how it works and where we need it...

We'll add more details. You don't need to use them though, they were compiled in our trusted setup ceremony. You only need the zk files generated after that ceremony: https://semaphore.appliedzkp.org/docs/glossary#trusted-setup-files
